sdk/lib/cmlib/hivewrt.c

```diff
@@ -135,7 +135,8 @@ HvpWriteLog(
      * Now calculate the bitmap and buffer sizes to hold up our
      * contents in a buffer.
      */
-    BitmapSize = ROUND_UP(sizeof(ULONG) + RegistryHive->DirtyVector.SizeOfBitMap / 8, HSECTOR_SIZE);
+    BitmapSize = ROUND_UP(sizeof(ULONG) + RegistryHive->DirtyVector.SizeOfBitMap, HSECTOR_SIZE);
+    ASSERT(BitmapSize == HV_LOG_GET_DIRTY_BITMAP_SIZE(RegistryHive->BaseBlock->Length));
```

Comment:
Changing this ASSERT to the following works for me.
```
    ASSERT(ROUND_UP(RegistryHive->DirtyVector.SizeOfBitMap, HSECTOR_SIZE) ==
           HV_LOG_GET_DIRTY_BITMAP_SIZE(RegistryHive->BaseBlock->Length));
```
I just leave out the addition of the 'sizeof(ULONG)' from what was there before.